docs/guides/data-loading.md

```diff
@@ -229,19 +229,23 @@ export default function Product() {
 ```
 
 ## Cloudflare KV
+If you picked Cloudflare Pages as your environment, [Cloudflare Key Value][cloudflare-kv] storage allows you to persist data at the edge as if it were a static resource. To start with local development, you need to add a `--kv` parameter with a name of your namespace to the package.json task, so it would look like this:
 
-If you picked Cloudflare Workers as your environment, [Cloudflare Key Value][cloudflare-kv] storage allows you to persist data at the edge as if it were a static resource. You'll need to [do some configuration][cloudflare-kv-setup] but then you can access the data from your loaders:
+`
+"dev:wrangler": "cross-env NODE_ENV=development wrangler pages dev ./public --kv PRODUCTS_KV"
+`
 
-```tsx filename=app/routes/products/$productId.tsx
-import type { LoaderFunction } from "@remix-run/node"; // or "@remix-run/cloudflare"
-import { json } from "@remix-run/node"; // or "@remix-run/cloudflare"
+This enable you to use the `PRODUCTS_KV` in a loader context:
```

Comment:
The context is initiated on line 245 and it is used on line 248. Pages for some reason require this, while Workers does not.

Reply:
It's probably better to just generalize this into something about "getLoadContext" is the place where you can pass CF env things through to your action / loader context so it can be applicable to Workers / Pages.

Reply:
Good point. Do we do the same thing with `context` for workers? You can get things off of `context` in that environment as well?

Reply:
Actually, fixing this behavior to have the same code for both platforms would be great. I consider the Pages code easier to understand though, context explains where the namespace comes from.